text/0000-issue-workflow.md

```diff
@@ -0,0 +1,222 @@
+  I  |                         |                    L  |                         |
+  S  |             +-----------v----------+            |             +-----------v----------+
+  S  |             |                      |         R  |             |                      |
+  U  |             |     status/ready     |         E  |             | status/code-approved |
+  E  |             |                      |         Q  |             |                      |
+  S  |             +----------------------+         U  |             +----------------------+
+     |                         |                    E  |                         |
+     |                         |                    S  |                         |
+     |             +-----------v----------+         T  |             +-----------v----------+
+     |             |                      |         S  |             |                      |
+     |             |  status/in-progress  |            |             | status/user-accepted |
+     |             |                      |            |             |                      |
+     +----------+  +----------------------+            +----------+  +----------------------+
+                               |                                                 |
+                               |                                                 |
+                               |                                                 |
+                               v                                                 v
+                             CLOSED                                            CLOSED
+```
+
+
+### Actionable Issues
+
+An actionable issue is one that can be worked on such that the following is true:
+
+- Definition of done - that may be acceptance criteria or clear expectations.
+- Appropriate labels
+    - `type/*`
+    - `size/*` (if possible)
+    - `status/ready`
+    - `good first issue` (if applicable)
+
+
+### Triage
+
+The following is a flow assist contributors in assigning the appropriate labels in script format.
+
+#### Issues
+
+- Is this a support request?
+    - Yes
+        - Add label type/support
+        - Provide support and close the issue with label `status/resolved`\
+          OR 
+        - Request further information from the issue author and label `status/incomplete`
+        - If activity is missing for 7 business days, follow up or close
+    - No
+        - See below
+- Is this something we want to do?
+    - I don’t even understand this issue
+        - Remove label `status/triage`
+        - Add label `status/invalid`
+        - Comment and close
+    - No - not valid / relevant for our project
+        - Remove label status/triage
+        - Add label `status/not-planned` or `status/invalid`
+        - Comment and close
+    - Maybe - needs RFC
+        - Remove label `status/triage`
+        - Add label `status/requires-rfc`
+        - Comment and close
```

Comment:
In practice we've tended to leave a comment to the effect of "this requires an RFC" and then wait some polite amount of time for the person to respond. Maybe it's worth updating the language here to account for that discretion.